src/libostree/ostree-repo.c

```diff
@@ -5081,7 +5081,8 @@ _ostree_repo_gpg_verify_data_internal (OstreeRepo    *self,
     }
   else if (remote_name != NULL)
     {
-      g_autofree char *gpgkeypath = NULL;
+      char **gpgkeypath_list = NULL;
```

Comment:
You can use `g_auto(GStrv)` for this and drop the `g_strfreev`s.

Reply:
Thanks! Avoids needing to `g_strfreev` before returning early. Addressed in https://github.com/ostreedev/ostree/pull/1773/commits/1e120d2a41107b9e128e1bed2820eceb080a69fb